test/system/035-logs.bats

```diff
@@ -91,6 +91,10 @@ ${cid[0]} d"   "Sequential output from logs"
 
 function _log_test_restarted() {
     run_podman run --log-driver=$1 --name logtest $IMAGE sh -c 'start=0; if test -s log; then start=`tail -n 1 log`; fi; seq `expr $start + 1` `expr $start + 10` | tee -a log'
+    # run/start is flaking for remote (see #9597) so let's
```

Comment:
Small favor request: could you phrase this as `FIXME: #9597` or `FIXME: ...blah blah #9597`? That way my `find-obsolete-skips` script will find it, and alert me when(if) 9597 gets fixed.

Reply:
Sure.